packages/ui/src/forum/hooks/useForumCategoryBreadcrumbs.ts

```diff
@@ -1,12 +1,12 @@
 import { useMemo } from 'react'
 
 import { useGetForumCategoryBreadcrumbsQuery } from '../queries'
-import { asForumBreadcrumbs } from '../types'
+import { asCategoryBreadcrumbs } from '../types'
 
-export const useForumBreadcrumbs = (id: string) => {
+export const useForumCategoryBreadcrumbs = (id: string) => {
   const { data, loading } = useGetForumCategoryBreadcrumbsQuery({ variables: { where: { id } } })
   const breadcrumbs = useMemo(
-    () => (data?.forumCategoryByUniqueInput ? asForumBreadcrumbs(data.forumCategoryByUniqueInput) : undefined),
+    () => (data?.forumCategoryByUniqueInput ? asCategoryBreadcrumbs(data.forumCategoryByUniqueInput) : undefined),
```

Comment:
In /proposals/preview/:id and /working-groups/openings/:id the breadcrumbs shows the id while loading. Maybe there could be a default a default:
```suggestion
    () => (data?.forumCategoryByUniqueInput ? asCategoryBreadcrumbs(data.forumCategoryByUniqueInput) : [{ id, title: `category ${id}` }]),
```
WDYT ?